site/docs/v1/tech/guides/api-authorization.adoc

```diff
@@ -0,0 +1,116 @@
+
+After the OAuth grant occurs, credentials are saved off, and then the Reminder App can present the stored JWT to the Todo API. If the JWT has expired, the Reminder App can present the refresh token to FusionAuth to retrieve a new JWT. 
+
+When presented with the token, the Todo API can verify it was signed by FusionAuth, possibly using JWKS to find the public key. It should validate the claims in the JWT, such as the expiration of the token, the issuer, and others. This processing has the benefit of requiring no contact between the Todo API and FusionAuth.
+
+The Todo API can also call the `/oauth2/token` endpoint with the JWT to check its validity.
+
+Strengths of this approach:
+
+* OAuth is a standard and thus may be more familiar to developers. There are also libraries to parse and validate JWTs.
+* This flow is similar to what large companies such as Facebook and Google require for access to their APIs.
+* Additional information can be stored in the JWT if needed (like roles).
+* The Todo API can check credential validity without communicating with FusionAuth. This helps with scale.
+* Logins are logged and could be used for auditing purposes.
+
+Challenges of this approach:
+
+* The process for gathering the credentials is more complex than other options.
+* A human being must be present for the initial credential grants.
+
+=== FusionAuth API Keys
+
+Another option is to use link:/docs/v1/tech/apis/authentication#api-key-authentication[FusionAuth API keys].
+
+To distribute the key, you would log in to the FusionAuth admin user interface, then create an API key and provide it to the Reminder App developer. 
+
+The Reminder App would present this API key to the Todo App every time they needed access. 
+
+To validate this credential, the Todo API would make a call against some FusionAuth API endpoint with the key. If the call succeeds, the Todo API returns the requested task data. If, instead, FusionAuth responds with a 401 response, the Todo API denies access.
+
+When using this approach, ensure the FusionAuth API key is narrowly scoped so if it is compromised, there are no negative implications beyond access to the Todo API. For example, you could grant read-only access to a FusionAuth feature you aren't using.
+
+Strengths of this approach:
+
+* There is only one secret to distribute per client application, and distribution is straightforward. The Reminder App developers get the API key one time and it is good forever.
+* There is no browser interaction required in the credentials distribution process.
+
+Challenges of this approach:
+
+* There is no easy way to automatically expire the credentials. FusionAuth API keys are valid until deleted. And such deletion is a manual process.
+* Similarly, there is no programmatic way to create these keys. A human being must log in to the admin UI.
```

Comment:
With the exception of Kickstart, but for this use case, kickstart probably doesn't apply.

Reply:
Good point, I will clarify that there's no programmatic way to add API keys other than at fusionauth instance initialization.